english/light-fire/graham-crackers/chocolate/find-chocolate.md

```diff
@@ -1,11 +1,2 @@
 You don't find any chocolate in your jacket, but you do find some pre-fermented and dried cocoa beans and some sugar.
 
-You realize that you could make your own chocolate out of these ingredients,
-but it would probably take a while. You take a moment to weigh your options.
-
-You decide to:
-
-[Be lazy](../eat-goo/eat-goo.md) (just eat the graham crackers and marshmallow goo already)
-
-[Be patient](make-chocolate/make-chocolate.md) (try to make chocolate)
-
```

Comment:
I took a look at your history, and making the separate pull request for each is not the problem. I believe the problem is the "Revert other changes" commit. Based on that commit, Git is concluding that you actually want to delete your other change.

I think the easiest way to fix this would be to create a new branch starting from the current master, then add only the commits you want to keep in that branch. You can add each commit you want using `git cherry-pick` (make sure to add them in the order you made them if there are multiple). Then you would need to close this pull request and make a new one.

Alternatively, you could run `git rebase -i` in your existing branch and use the resulting interface to choose which commits to keep. After this, you'll need to run a force push with `git push -f`, since you want to remove commits from the remote repository, which isn't allowed by default. I think this strategy will be a bit trickier to get right.

Either way, this situation goes beyond what the course covered. Sorry you're having this trouble.